src/main/java/gg/solarmc/solarcredits/command/commands/SpendCommand.java

```diff
@@ -0,0 +1,123 @@
+package gg.solarmc.solarcredits.command.commands;
+
+import com.google.gson.JsonParser;
+import gg.solarmc.loader.credits.CreditsKey;
+import gg.solarmc.loader.credits.WithdrawResult;
+import gg.solarmc.solarcredits.SolarCredit;
+import gg.solarmc.solarcredits.command.CommandHelper;
+import gg.solarmc.solarcredits.command.CreditSubCommand;
+import gg.solarmc.solarcredits.menus.ConfirmMenu;
+import okhttp3.FormBody;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.function.Consumer;
+
+public record SpendCommand(SolarCredit plugin, String X_TEBEX_SECRET) implements CreditSubCommand {
```

Comment:
```suggestion
public record SpendCommand(SolarCredit plugin, String tebexSecret) implements CreditSubCommand {
```
In general, fields should always be lowerCamelCase. Reserve UPPER_SNAKE_CASE for constants